src/main/java/com/sem/btrouble/model/GamePowerUp.java

```diff
@@ -14,6 +15,7 @@
  * @author Martin
  *
  */
+@SuppressWarnings("serial")
```

Comment:
I don't get serial warnings in IntelliJ. Maybe it is Eclipse specific? 
Though, we could fix this by not extending these classes from the Slick2D Shape class, which implements Serializable(which is the cause of this warning).